.github/workflows/ci.yml

```diff
@@ -23,38 +35,17 @@ jobs:
     - name: Install Playwright browsers
       run: |
         npx playwright install --with-deps
-    - name: Start test environment
+    - name: Find all Jupyter notebooks
+      run: find .. -name '*.ipynb' | sort > ./files.txt
+    - name: Start test environments
       id: start-test-environment
-      run: |
-        RESULT=$(curl --location --request POST 'https://api.clearscape.teradata.com/environments' \
-          --header "Authorization: Bearer $CSAE_TOKEN" \
-          --header 'Content-Type: application/json' \
-          --data-raw "{
-              \"name\": \"$CSAE_ENV_NAME\",
-              \"region\": \"us-central\",
-              \"password\": \"$CSAE_ENV_PASSWORD\"
-          }")
-        # replace with jq
-        JUPYTER_ENV=$(echo $RESULT | grep url | head -n 1 | cut -d"\"" -f 4)
-        echo "jupyter-env-url=$JUPYTER_ENV" >> $GITHUB_OUTPUT
-      env:
-        CSAE_TOKEN: '${{ secrets.CSAE_TOKEN }}'
-        CSAE_ENV_PASSWORD: '${{ secrets.CSAE_ENV_PASSWORD }}'
-        CSAE_ENV_NAME: '${{ steps.define-environment-name.outputs.env-name}}'
+      run: npx ts-node pre_create_env.ts
     - name: Run Playwright tests
       run: |
         npx playwright test
-      env:
-        ENVIRONMENT: ${{ steps.start-test-environment.outputs.jupyter-env-url }}
-        CSAE_ENV_PASSWORD: '${{ secrets.CSAE_ENV_PASSWORD }}'
-    - name: Delete test environment
+    - name: Delete test environments
       if: always()
-      run: |
-        curl --location --request DELETE "https://api.clearscape.teradata.com/environments/$CSAE_ENV_NAME" \
-          --header "Authorization: Bearer $CSAE_TOKEN"
-      env:
-        CSAE_TOKEN: '${{ secrets.CSAE_TOKEN }}'
-        CSAE_ENV_NAME: '${{ steps.define-environment-name.outputs.env-name}}'        
+      run: npx ts-node post_delete_env
```

Comment:
don't you need `.ts` suffix here as well the same way you have it on line 42: `pre_create_env.ts` ?

Reply:
Good catch btw I some how overlooked.
it worked, it assumes I think. 
But I will add it for consistency.